doc/source/cluster/kubernetes/examples/gpu-training-example.md

```diff
@@ -0,0 +1,149 @@
+(kuberay-gpu-training-example)=
+
+# ML training with GPU on Kubernetes
+In this guide, we show you how to run a sample Ray machine learning training workload with GPU on Kubernetes infrastructure. We will run Ray's {ref}`PyTorch image training benchmark <pytorch_gpu_training_benchmark>` with a 1 gigabyte training set. The following script consists of:
```

Comment:
My natural inclination would be to put the end-to-end script at the end, as a summary.

On the other hand, the approach you've taken is also quite nice -- it presents the necessary info upfront for users who just need the quick reference.

I'd suggest placing the script in its own section, titled something like 

```
## The end-to-end workflow 
```
Then introduce the section with something like
```
The following script summarizes the end-to-end workflow for GPU training.
These instructions are for GCP, but a similar setup would work for any major cloud provider.
The script consists of
- Step 1:...
```
Then paste the script.

Then state something like:
```
In the rest of this document, we present a more detailed breakdown of the above workflow.
```

Then the next section with
```
## Step 1: Set up a Kubernetes cluster on GCP.
```

Reply:
Updated.